e2e/testdata/fn-render/invalid-inline-fnconfig/.expected/config.yaml

```diff
@@ -12,4 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-exitCode: 1
\ No newline at end of file
+exitCode: 1
+stdErr: "missing Resource metadata"
```

Comment:
ha ha ... this is terrible error message. Poor user has to go hunt....which metadata :)

Reply:
The test harness is checking for a substring - the actual error output is much longer. I can update the PR to show the entire error message here